test/functional/feature_cltv.py

```diff
@@ -60,6 +60,7 @@ def set_test_params(self):
             '-dip3params=9000:9000',
             '-par=1',  # Use only one script thread to get the exact reject reason for testing
             '-acceptnonstdtxn=1',  # cltv_invalidate is nonstandard
+            '-vbparams=v20:0:999999999999:480:384:288:5:-1' # Delay v20 for this test as we don't need it
```

Comment:
note for reviewers: if this PR would be merged after https://github.com/dashpay/dash/pull/5658 need to replace -1 to 0 so far as MNActivationHeight is refactored to flag useEHF.
```suggestion
            '-vbparams=v20:0:999999999999:480:384:288:5:0' # Delay v20 for this test as we don't need it
```

Reply:
@knst Since this one was merged, adjust `feature_cltv.py` on #5658